src/transactions/feed/JumpstartTransactionDetailsScreen.tsx

```diff
@@ -249,13 +339,12 @@ function JumpstartTransactionDetailsScreen({ route }: Props) {
           copySuccessMessage={t('walletConnectRequest.transactionDataCopied')}
           testID="JumpstarReclaimBottomSheet/RequestPayload"
         />
-        {reclaimTx && (
-          <EstimatedNetworkFee networkId={transaction.networkId} transaction={reclaimTx} />
-        )}
-        {/* TODO: Implement confirm */}
+        {reclaimTx && <EstimatedNetworkFee networkId={networkId} transaction={reclaimTx} />}
         <Button
           text={t('confirm')}
-          onPress={() => Logger.debug('confirmed')}
+          showLoading={reclaimStatus === 'loading'}
+          disabled={reclaimStatus !== 'idle'}
```

Comment:
```suggestion
          disabled={reclaimStatus === 'loading' || reclaimStatus === 'success'}
```

Reply:
Hmm, why?
I think we don't want to show the button enabled when there was an error. So it should only be enabled if it's in `idle` state.

Reply:
ah, i think this also relates to our discussion about the refetch above. i was thinking that it is a more obvious pattern to press the "reclaim" again to trigger the reclaim, so when there is an error, the button should be enabled again. if we think that is a good pattern, then it also removes the need for the error message to trigger the claim (but we can do it from there too, i feel more strongly that a user should be able to reclaim by pressing the reclaim)